doc/implementers-guide/Makefile.am

```diff
@@ -5,7 +5,7 @@ TARGET = implementers-guide.html
 
 EXTRA_DIST = $(SRC)
 
-doc_DATA = $(TARGET)
+doc_DATA = $(top_srcdir)/doc/implementers-guide/$(TARGET)
```

Comment:
@lumag no, it does not work https://travis-ci.org/muvarov/odp/jobs/247575518 also the same full patch is for other docs.